docs/UserGuide.md

```diff
@@ -175,8 +176,7 @@ You can use `Start Tags` to mark the start of your contribution. The author spec
 
 #### Standalone Configuration
 
-Instead of self-configuring all the repository details, you can have the repository owners include a 
-`Standalone Configuration` to maintain their own repository configuration. For more information, check out the
+Instead of self-configuring all the repository details, you can have the repository owners include a `Standalone Configuration` to maintain their own repository configuration. For more information, check out the
```

Comment:
why this change?

Reply:
Oh, was just cleaning up the documents, since there was trailing whitespace and a newline there, which wasn't necessary (my travis was failing earlier because of the `run_check.sh` I think, so I thought to clear those warnings as well since I'm editing the docs anyway)

Reply:
Use _Standalone Configuration_ instead of `Standalone Configuration` (because this is simply a new term, not a reference to something in the code)